packages/tools/generator-pie-component/src/app/templates/test/accessibility/pie-skeleton.__spec__.ts

```diff
@@ -0,0 +1,22 @@
+
+import { test, expect } from '@sand4rt/experimental-ct-web';
+import AxeBuilder from '@axe-core/playwright';
+import { Pie<%= componentName %>, <%= componentName %>Props } from '@/index';
+
+test.describe('Pie<%= componentName %> - Accessibility tests', () => {
+    test('a11y - should test the Pie<%= componentName %> component WCAG compliance', async ({ page, mount }) => {
+        await mount(
+            Pie<%= componentName %>,
+            {
+                props: {} as <%= componentName %>Props,
+            },
+        );
+
+        const results = await new AxeBuilder({ page })
+        .withTags(['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria'])
+        .disableRules(['color-contrast', 'color-contrast-enhanced'])
```

Comment:
I wonder if these could be variables in a shared config we pull in? Saves a headache in future if we alter the a11y test setup

Reply:
how about moving it to` packages/components/pie-webc-testing/src/helpers/defs.ts`?

Reply:
the `defs` pattern is more for defining typescript types, I wonder if we could export constants from the playwright config package/folder we already have?

Reply:
Refactoring this code has actually been something I've wanted to do for a while, but haven't got around to creating a ticket for.

Ideally, we should be utilising Playwright fixtures, so that all of the axeBuilder config is defined in a single place.

You can see an example here with PIE Docs. 

Playwright Fixture - https://github.com/justeattakeaway/pie/blob/main/apps/pie-docs/test/playwright/fixtures.js#L9-L16

Usage in test spec - https://github.com/justeattakeaway/pie/blob/main/apps/pie-docs/test/accessibility/accessibility.spec.js#L18-L20

When this does eventually get refactored, these fixtures should probably be moved into the pie testing package.

Reply:
I just refactored to implement @jamieomaguire's suggestion, do you reckon this should be done part of this PR?

Reply:
Nah, happy for this to be covered as part of another ticket.

Reply:
added this as part of this ticket